### PR TITLE
V0 Two Piece back panel with vents

### PR DESCRIPTION
**Pull request by Polar-Ted**
* [x] The mod, firmware configuration or slicer profile is in the correct category
    folder. Printable mods go to `printer_mods/`, firmware configurations
    go to `firmware_configurations/`, slicer profiles go to `slicer_profiles/`.
    Create a subfolder with your name, and place the mods in a subfolder with
    a descriptive name within that folder, e.g.: `/printer_mods/FHeilmann/flux_capacitor`
  * [x] Folders names MUST NOT contain spaces. If possible, make sure file names also 
    do not contain any spaces.
  * [x] For each mod, add a small `README.md` file to its folder with a short description
    of what the mod accomplishes. This readme can be used to add pictures, give assembly
    instructions or specify a bill of materials if the mod requires additional hardware.
  * [x] The PR modifies the top-level `README.md` of the category folder adding the 
    contribution to the table. Read the top part of the file for instructions on how
    to do this. Please preserve the alphabetical ordering while adding new rows. Make sure
    to fill out the compatibility matrix to indicate which versions of the Voron printer
    the submission is compatible with.
  * [x] The mod/configuration/profile has been tested by the person submitting the mod 
    and/or other Voron users. Make sure to add information about how the mod was tested below. 
  * [x] The mod is not merely a slight modification of an official Voron part, configuration 
    or profile (i.e. an official Voron part with a few mm added or removed or a slicer profile 
    which only modifies a few values). *(When in doubt, contact one of the admins in the 
    Voron discord before submitting the PR)*
  * [x] Submitted STLs are printable without support. *(If the mod does not meet this criterion
    join the Voron discord and ask the other users for advice on how to modify the mod such 
    that it does not require supports)*
  * [x] Submitted STL files are not corrupt. *(This can be tested by opening the STL in PrusaSlicer
    and checking if mesh errors are reported.)*
  * [x] Submitted STL files are oriented and scaled properly for printing.
  * [x] Submission includes a CAD file in the form of a `.STEP` or `.SCAD` file
  * [x] Submitted firmware configs or slicer profiles contain no sensitive data (e.g. API keys).

#### Which mods/configurations/profiles are added by this PR?
Printed two part Back Panel for V0 with vents. 

#### How was it tested? 
Printed and in use on V0.1781

#### Any background context you want to provide?
Created to aid cooling while protecting electronics.

### CODE REVIEW

File: printer_mods/README.md [hunk 1]
```diff
@@ -294,6 +294,7 @@ like so:
 |padok |[Chain Anchor - ADXL345 mount Mod](./padok/chain_anchor-ADXL345_mount) | A ADXL345 mount for the Afterburner |:x: |:grey_question: |:heavy_check_mark: |:grey_question: |
 |Phalanx |["Other" Idlers Mod](./Phalanx/Other-V2-Idlers/) | Allow usage of wider toolheads on the V2 |:x: |:grey_question: |:heavy_check_mark: |:x:|
 |piet206 |[V1.8 Z Rails Mod](./piet206/V1.8._Z_Rails_Mod) | Add MGN12H linear rails for Z axis motion |:x: |:heavy_check_mark: |:x: |:x:|
+|Polar-Ted|[V0.1 Two Piece Back Panel](./Polar-Ted/V0_Two_Piece_Back_Panel/) | Vented 2 piece back panel for V0 |:heavy_check_mark: |:x: |:x: |:x:|
```
Comment (cbc02009) [inline printer_mods/README.md:297]: kinda nitpicky, but this should go after Polared's mods

**Review by cbc02009 — CHANGES_REQUESTED** [review]

File: printer_mods/README.md (hunk 1 above)
Comment (Polar-Ted) [reply printer_mods/README.md]: Done

**Comment by MapleLeafMakers** [comment]
@Polar-Ted Can you delete the empty README.md files in the STEP, STL and Images directories?

**Comment by Polar-Ted** [comment]
> @Polar-Ted Can you delete the empty README.md files
> 
> * printer_mods/Polar-Ted/README.md
> * printer_mods/Polar-Ted/V0_Two_Piece_Back_Panel/Images/README.md
> * printer_mods/Polar-Ted/V0_Two_Piece_Back_Panel/STEP/README.md
> * printer_mods/Polar-Ted/V0_Two_Piece_Back_Panel/STL/README.md

Done

File: printer_mods/README.md (hunk 1 above)
Comment (Polar-Ted) [reply printer_mods/README.md]: Requested Change was made 10 days ago

**Review by cbc02009 — APPROVED** [review]

**Pull request merged**